instances/measuredSignalType/alphaActivity.jsonld

```diff
@@ -0,0 +1,24 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/measuredSignalType/alphaActivity",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/measuredSignalType",
+  "definition": "Alpha activity is an electromagnetic oscillation in the frequency range of 8–12 Hz arising from synchronous and coherent (in phase / constructive) electrical activity. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Alpha_wave)]",
```

Comment:
I rather suggest the following:

"Alpha activity is a neural oscillation in the low frequency range (typically between 8-12 Hz) arising from synchronous and coherent electrical activity in the brain. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Alpha_wave)]"

@apdavison would you agree with this definition?